judge/fixtures/language_all.json

```diff
@@ -942,5 +942,21 @@
         "description": "",
         "extension": ""
     }
+},
+{
+    "model": "judge.language",
+    "pk": 75,
+    "fields": {
+        "key": "ZIG",
+        "name": "Zig",
+        "short_name": "",
+        "common_name": "Zig",
+        "ace": "zig",
+        "pygments": "zig",
+        "template": "",
+        "info": "zig 0.6.0",
```

Comment:
Don't override the `info` field unless the judge misreports the version itself.

Reply:
I'm assuming just version (`0.6.0`)  and not a name is valid~